bootstrap/kubeadm/config/manager/manager.yaml

```diff
@@ -22,6 +22,7 @@ spec:
         - "--leader-elect"
         - "--metrics-bind-addr=localhost:8080"
         - "--feature-gates=MachinePool=${EXP_MACHINE_POOL:=false},KubeadmBootstrapFormatIgnition=${EXP_KUBEADM_BOOTSTRAP_FORMAT_IGNITION:=false}"
+        - "--bootstrap-token-ttl=${KUBEADM_BOOTSTRAP_TOKEN_TTL:=0}"
```

Comment:
missed it the first time, I think we have to set the default to the same as the default of the flag which is "15m". I assume otherwise we have 0 if the env var is not set

Reply:
TL;DR You're right, I failed to appreciate the CLI arg was a `Duration` as opposed to int - will update.

Originally I thought the CLI arg was an `Int` and figured I would need time in nanoseconds. Consequently I leaned into the default behavior, when `0` is used, defined in the `SetupWithManager`.

Reply:
Fixed.